### ISSUE TITLE
Styling radio inputs

### ISSUE THREAD

**Issue by EsaMakinen** [futurice/symptomradar#93]

**Comment by Doniee** [comment]
Accessibility should be ensured if do this. Maybe something like this https://www.a11ywithlindsey.com/blog/create-custom-keyboard-accessible-radio-buttons
